src/app/modules/cerf/cerf.component.html

```diff
@@ -148,7 +148,7 @@ <h3 class="mat-subheading-1">Event Tags*</h3>
 
 		<div id="tags">
 			<div *ngFor="let tag of tagIds; let i=index">
-				<mat-checkbox [checked]="cerfForm.get('tags').value.includes(tag)" [disabled]="cerfForm.disabled" (change)="addRemoveTag(tag, $event.checked)">{{ tagNames[i] }}</mat-checkbox>
+				<mat-checkbox [checked]="cerfForm.get('tags').value.includes(tag)" [disableControl]="cerfForm.disableControl" (change)="addRemoveTag(tag, $event.checked)">{{ tagNames[i] }}</mat-checkbox>
```

Comment:
Concerning the last paragraph on this: https://trungk18.com/experience/disable-a-reactive-form-control-using-custom-directive/
Please check that the check marks on tags will not disappear due to .value() no longer returning what is expected when disabled. Also, more importantly, that dispatchCerf() (or any method that relies on .value()) still works when the cerf form is disabled. You may have to fix these instances. Let me know if you need help with checking those, I'm not sure if tags are fully working yet in the first place...